### DISCUSSION TITLE
Welcome to QGreenland 2023 Researcher Workshop Discussions!

### DISCUSSION THREAD

**Discussion by MattF-NSIDC** [qgreenland-workshop-2023-researcher/qgreenland-workshop-2023-researcher.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alysethurber** [comment]
Hi! I'm Alyse. I try to add gifs to discussion threads.

**Comment by MattF-NSIDC** [comment]
```
![Silly ice gif](https://i.giphy.com/media/KaW6fNYZf6eSk/giphy.gif)
```

![Silly ice gif](https://i.giphy.com/media/KaW6fNYZf6eSk/giphy.gif)
